enlighten/measurement/SaveOptions.py

```diff
@@ -211,6 +214,11 @@ def init_from_config(self):
         self.le_suffix          .setText   (self.config.get(s, "suffix"))
         self.le_note            .setText   (re.sub(",", "", self.config.get(s, "note")))
 
+        self.directory = self.ctl.config.get("user", "save_location", default=common.get_default_data_dir())
```

Comment:
I would have chosen "SaveOptions" as the section rather than "user" (to make it easy to tell which code/class owns/controls which settings), but that's fine.